tests/brevitas/core/bit_width_fixture.py

```diff
@@ -57,6 +59,15 @@ def bit_width_const(bit_width_init):
     return BitWidthConst(bit_width_init)
 
 
+@pytest_cases.fixture()
+def bit_width_stateful_const(bit_width_init):
```

Comment:
I would add an `bit_width_init_two` at the bottom of the file to duplicate the bit_width_init fixture by setting `bit_width_init_two = bit_width_init` (as done here https://stackoverflow.com/a/39444098), and then use it here.